packages/remote-feature-flag-controller/src/remote-feature-flag-controller.test.ts

```diff
@@ -41,7 +41,7 @@ const MOCK_FLAGS_WITH_THRESHOLD = {
   ],
 };
 
-const MOCK_METRICS_ID = '0x1234567890abcdef';
+const MOCK_METRICS_ID = 'f9e8d7c6-b5a4-3210-9876-543210fedcba';
```

Comment:
Not a valid UUIDv4, this is UUIDv3

Reply:
Good catch ! addressed 93d835110a7089682e01780d465f1ae1f6556332